### PR TITLE
Added aliases and fixed :loader: symbols

### PR DESCRIPTION
**Pull request by freza1**
makes it easier to play music

### CODE REVIEW

File: Vita3KBot/Commands/Music.cs [hunk 1]
```diff
@@ -244,12 +247,13 @@ public async Task SkipTask()
...
 
         [Group("play"), InVoiceChannel]
+        [Alias("p")]
         public class Play : InteractiveBase {
             [Command(RunMode = RunMode.Async), Name("play")]
             [Summary("Plays a song from tong of sources")]
             public async Task PlayTask([Remainder] string query)
             {
-                var final = await ReplyAsync("<a:loader:461159122575032331> Searching");
+                var final = await ReplyAsync("<a:face_with_monocle:461159122575032331> Searching");
```
Comment (IllusionMan1212) [inline Vita3KBot/Commands/Music.cs:256]: can you replace the verbose emoji code with just a simple 🧐

File: Vita3KBot/Commands/Music.cs [hunk 2]
```diff
@@ -396,7 +400,7 @@ public async Task PlayTask([Remainder] string query)
     public class MusicModule {
         public static async Task PlayNextTrack(TrackEndedEventArgs args) {
             var player = args.Player;
-            var final = await player.TextChannel.SendMessageAsync("<a:loader:461159122575032331> Playing the next song");
+            var final = await player.TextChannel.SendMessageAsync("<a:face_with_monocle:461159122575032331> Playing the next song");
```
Comment (IllusionMan1212) [inline Vita3KBot/Commands/Music.cs:403]: same thing here

**Review by IllusionMan1212 — APPROVED** [review]
LGTM, if you could do those simple changes that would be nice 👌

File: Vita3KBot/Commands/Music.cs (hunk 1 above)
Comment (freza1) [reply Vita3KBot/Commands/Music.cs:256]: got it. ill see whether that's actually possible with my keyboard.

**Comment by freza1** [comment]
done. that SHOULD be the last of it.

**Pull request merged**